packages/react/src/components/Button/Button.tsx

```diff
@@ -21,66 +18,55 @@ export type ButtonProps = {
    */
   size?: 'sm' | 'md' | 'lg';
   /**
-   * If `Button` should fill full width of its container
+   * If `Button` should fill width of its container
    * @default false
    */
-  fullWidth?: boolean;
-  /** Toggle icon only styling, pass icon as children
+  fill?: boolean;
+  /** Toggle loading state
    * @default false
    */
-  icon?: boolean;
```

Comment:
Ref comment above, we need this for cases where the icon is not the only child in a button

Reply:
Ikke lengre behov da ikon har negativ margin - gått gjennom med @Thunear og @Febakke :)

Reply:
Sjå https://github.com/digdir/designsystemet/pull/2313#discussion_r1743244945

Reply:
Resolved by re-introducing icon prop